wgpu-types/src/lib.rs

```diff
@@ -793,10 +793,18 @@ bitflags::bitflags! {
         /// This is a native-only feature.
         const RAY_TRACING_ACCELERATION_STRUCTURE = 1 << 56;
 
-        // 57 available
-
         // Shader:
 
+        /// Enables support for debugPrintf in WGSL shaders.
+        ///
+        /// Supported Platforms:
+        /// - DX11 (fxc only)
+        /// - DX12 (fxc only)
+        /// - Vulkan
+        /// - OpenGL
```

Comment:
We don't support it on GL, so we should label this as an "unimplemented platform, if it is implementable